SPECS/cmake/cmake.spec

```diff
@@ -21,6 +21,7 @@ Patch6:         CVE-2023-38545.patch
 Patch7:         CVE-2023-38546.patch
 Patch8:         cve-2023-44487.patch
 Patch9:         CVE-2023-28320.patch
+Patch10:        CVE-2023-27534.patch
```

Comment:
what about CVE-2024-27533?  Your 2 PR's should be combined.

Reply:
Thanks @jslobodzian. I have cherry-picked the changes of CVE-2024-27533 PR #10467 into this PR itself. Please review it.